src/tensors/gpu/swap.cu

```diff
@@ -0,0 +1,13 @@
+#include "cuda_helpers.h"
+#include "swap.h"
+void copyCpuToGpu(const char * in, char * gpuOut);
+void copyGpuToGpu(const char * in, char * gpuOut);
+
+namespace marian {
+    namespace swapper {
```

Comment:
Nit: please reformat these two files following the general code conventions that we seem to use: indentation via 2 spaces, no indentation within namespaces.

Reply:
Done